Overview.md

```diff
@@ -13,6 +13,9 @@ interacting with the stage (index).
 ## [basic-branching](basic-branching/README.md)
 The first stride into branching.
 
+## [basic-stashing](basic-stashing/README.md)
+The first stride into branching.
```

Comment:
This should probably be the first strige into _stashing_ Now I'm getting worried I solve the merge conflict wrong :)

Reply:
my bad